### PR TITLE
feat: Create new connection invitation

### PR DESCRIPTION
**Pull request by bhavanakarwade**
### What
- Modify the create invitation function, So that it checks if the Organization logo and Organization name have been updated, then a new connection invitation will be created.

### Why
- This modification ensures that invitations accurately reflect the current changes of the organization. If the organization's logo or name changes, it's essential that invitations reflect these updates.

### CODE REVIEW

File: apps/organization/src/organization.service.ts [hunk 1]
```diff
@@ -250,7 +250,7 @@ export class OrganizationService {
 
       const organizationExist = await this.organizationRepository.checkOrganizationExist(updateOrgDto.name, orgId);
```
Comment (ankita-p17) [inline apps/organization/src/organization.service.ts:251]: you can optimize this code
1. get org by name
2. check if org.Id received is same as orgId, if true, throw new ConflictException(ResponseMessages.organisation.error.exists);

**Review by ankita-p17 — CHANGES_REQUESTED** [review]

File: apps/organization/src/organization.service.ts [hunk 2]
```diff
@@ -268,7 +268,18 @@ export class OrganizationService {
         delete updateOrgDto.logo;
       }
 
-      const organizationDetails = await this.organizationRepository.updateOrganization(updateOrgDto);
+      let organizationDetails;
+      const checkAgentIsExists = await this.organizationRepository.getAgentInvitationDetails(orgId);
+
+      if (!checkAgentIsExists?.connectionInvitation && !checkAgentIsExists?.agentId) {
```
Comment (ankita-p17) [inline apps/organization/src/organization.service.ts:274]: Below these two if conditions to optimize the code and avoid checking same condition repeatedly

**Review by ankita-p17 — CHANGES_REQUESTED** [review]

File: apps/organization/src/organization.service.ts (hunk 1 above)
Comment (bhavanakarwade) [reply apps/organization/src/organization.service.ts]: Resolved

File: apps/organization/src/organization.service.ts (hunk 2 above)
Comment (bhavanakarwade) [reply apps/organization/src/organization.service.ts:275]: Resolved

**Review by ankita-p17 — APPROVED** [review]

**Pull request merged**